components/infobox/commons/infobox_team.lua

```diff
@@ -48,18 +49,27 @@ end
 function Team:createInfobox()
 	local infobox = self.infobox
 	local args = self.args
-	-- Need links in LPDB, so declare them outside of display code
+
+	--- Transform data
+	-- Links
 	local links = Links.transform(args)
 
+	-- Team Information
 	local team = args.teamtemplate or self.pagename
 	self.teamTemplate = mw.ext.TeamTemplate.teamexists(team) and mw.ext.TeamTemplate.raw(team) or {}
 
+	args.imagedark = args.imagedark or args.imagedarkmode or args.image or self.teamTemplate.imagedark
+	args.image = args.image or self.teamTemplate.image
+	args.teamcardimagedark = self.teamTemplate.imagedark or args.teamcardimagedark or args.teamcardimage
+	args.teamcardimage = self.teamTemplate.image or args.teamcardimage
+
+	-- Display
 	local widgets = {
 		Header{
 			name = args.name,
-			image = args.image,
+			image = not Game.isDefaultTeamLogo{logo = args.image} and args.image or nil,
```

Comment:
what if wikis want to show the default logos? (not that i know of any but maybe we should check upon that)

Reply:
`imageDefault` will still be used. So current behavior is kept